src/index.html

```diff
@@ -1,12 +1,174 @@
+            <p class="tag">drink</p>
+            <p class="tag">snack</p>
+          </div>
+        </section>
+        <section class="recipe-card-display center-column">
+          <i class="far fa-bookmark fa-4x"></i>
+          <img src="https://spoonacular.com/recipeImages/741603-556x370.jpeg"/>
+          <h4>Banana Pancakes</h4>
+          <div class="tag-box">
+            <p class="tag">breakfast</p>
+            <p class="tag">fruit</p>
+          </div>
+        </section>
+      </section>
+      <section class="recipe-cards fav-recipes-display hidden">
+        <!-- <h4>Favorite Recipes</h4> -->
+        <section class="recipe-card-display center-column">
+          <i class="fas fa-bookmark fa-4x"></i>
+          <img src="https://spoonacular.com/recipeImages/741603-556x370.jpeg"/>
+          <h4>Banana Pancakes</h4>
+          <div class="tag-box">
+            <p class="tag">breakfast</p>
+            <p class="tag">fruit</p>
+          </div>
+        </section>
+        <section class="recipe-card-display center-column">
+          <i class="fas fa-bookmark fa-4x"></i>
+          <img src="https://spoonacular.com/recipeImages/741603-556x370.jpeg"/>
+          <h4>Banana Pancakes</h4>
+          <div class="tag-box">
+            <p class="tag">breakfast</p>
+            <p class="tag">fruit</p>
+          </div>
+        </section>
+        <section class="recipe-card-display center-column">
+          <i class="fas fa-bookmark fa-4x"></i>
+          <img src="https://spoonacular.com/recipeImages/741603-556x370.jpeg"/>
+          <h4>Banana Pancakes</h4>
+          <div class="tag-box">
+            <p class="tag">breakfast</p>
+            <p class="tag">fruit</p>
+          </div>
+        </section>
+      </section>
+    </main>
+    <hr />
+    <hr />
+    <!-- <section class="box-outline center-column">
+      <h1>Pantry</h1>
+      <article class="list">
+        <div>
+          <input type="checkbox" id="item1" />
+          <label for="item1">flour</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">flour</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">flour</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">flour</label><br />
+        </div>
+      </article>
+    </section> -->
+        <div id="modal-1" aria-hidden="true">
 
+          <!-- [2] -->
+          <div tabindex="-1" data-micromodal-close>
+
+            <!-- [3] -->
+            <div role="dialog" aria-modal="true" aria-labelledby="modal-1-title" >
+
+
+              <header>
+              <h2 id="modal-1-title">
+                Modal Title
+              </h2>
+
+              <!-- [4] -->
+              <button aria-label="Close modal" data-micromodal-close></button>
+            </header>
+
+            <div id="modal-1-content">
+              Modal Content
+            </div>
+        </div>
+      </div>
+    </div>
```

Comment:
We can talk about this together today and Friday to determine if micromodal will be effective moving forward to display information about each recipe